src/main/java/be/acara/events/controller/EventController.java

```diff
@@ -45,13 +44,16 @@ public ResponseEntity deleteEvent(@PathVariable("id") long id) {
         return ResponseEntity.ok(eventService.getAllCategories());
     }
 
-    @PostMapping
+    @PostMapping("/new-event")
```

Comment:
Unnecessary, posting to /events should automatically mean creating a new event object.